connectors/package.json

```diff
@@ -8,10 +8,12 @@
     "build": "tsc",
     "start-api": "tsx ./src/api_server.ts -p 3002",
     "slack-worker": "tsx ./src/connectors/slack/worker.ts",
+    "notion-worker": "tsx ./src/connectors/notion/temporal/worker.ts",
```

Comment:
please put it in the same worker as slack and rename worker (bonus point if you put everything in the same binary with the api_server) :)

Reply:
[7e8abdd845645b24d05e07fcb6652fc3e96732f6https://github.com/dust-tt/dust/pull/251/commits/7e8abdd845645b24d05e07fcb6652fc3e96732f6](https://github.com/dust-tt/dust/pull/251/commits/7e8abdd845645b24d05e07fcb6652fc3e96732f6)